### ISSUE TITLE
csv allows empty titles

### ISSUE THREAD

**Issue by vanatteveldt** [amcat/amcat#666]
when uploading a csv, if a column used for title is empty it uploads fine, but on querying I get the error "KeyError: title"

```
 File "/home/amcat/amcat/amcat/tools/amcates_queryset.py", line 605, in <dictcomp>
     field: hit["highlight"][field] for field in fields
KeyError: 'title'
```

Either the query needs to be more robust, or the csv uploader needs to not allow empty values (or add a dummy value), or both

**Comment by CJStuart** [comment]
That's really strange. Title is a required non-blank field in the Django model. It shouldn't even validate.

**Comment by CJStuart** [comment]
Fixed in b9cbfcdc6241ffea40fad3552aef91aeba89838e
